pylint/checkers/typecheck.py

```diff
@@ -507,6 +507,8 @@ def _emit_no_member(
                 return False
             if metaclass:
                 # Renamed in Python 3.10 to `EnumType`
+                if _enum_has_attribute(owner, node):
+                    return False
```

Comment:
```suggestion
                if _enum_has_attribute(owner, node):
                    return False
                # Renamed in Python 3.10 to `EnumType`
```
I think the comment we related to line 512.

Reply:
Ta